my_param: 1000
image_topic: /kinect2/qhd/image_color